docs/articles/posts/20241231_llm_as_a_judge.md

```diff
@@ -0,0 +1,89 @@
+
+This brings us on to the star of the show: **LLM-as-a-Judge**. We want an evaluation method that understands the intricacies of language and is significantly quicker than humans.
+
+<figure markdown>
+![Example schema for a LLM judging the fluency of a piece of text.](../../images/LLM_as_a_Judge/prompt.png)
+<figcaption>Figure 2: This is an example schema used for an LLM judge that evaluates the fluency of a review. Fluency can be measured with only the summary; no additional information is needed. This is a shortened example, note that there are gaps where additional context would be, such as the example summary and the list of guidance.  </figcaption>
+</figure>
+
+An LLM-as-a-Judge uses an LLM to score certain characteristics of a summary. Typically, each characteristic is numerically scored based on instructions set out in a schema, and these characteristics can include groundedness, cohesiveness and relevance – metrics that can also be measured by humans.
+
+What's great about LLM-as-a-Judge is that you can score on characteristic you’d like. Figure 2 shows an example a schema written to measure a new characteristic: fluency.
+
+LLM-as-a-Judge is scientifically grounded. It has been shown that LLM judgements can [align to human feedback](https://arxiv.org/abs/2306.05685), and there are various [articles](https://huggingface.co/learn/cookbook/llm_judge) on how to improve you LLM judges.
+
+## Who judges the judges?
+
+Whilst LLM judges have been shown to be comparable to humans, we still need to be aware of the mistakes that LLMs can make. First, LLMs are typically non-deterministic, meaning an LLM can vary the score it gives a summary if ran multiple times. Secondly, LLM’s are known to hallucinate, so could give an entirely incorrect score.
+
+Whilst human judges can be self-aware of their limitations and effectively self-manage themselves implicitly, our LLM-as-a-judge may not have the self-awareness to identify when it’s not up to the task. In current literature three main approaches appear to be taken to evaluate the evaluators.
+
+1. **Comparison to Benchmark:** Several studies (see [1](https://arxiv.org/pdf/2406.18403), [2](https://arxiv.org/pdf/2310.07641) and [3](https://arxiv.org/pdf/2306.05685)) have explored using established benchmarks to compare the judges to.  However, all highlight various limitations of the general benchmarks and so this approach is not recommended as a final evaluation but can be useful in earlier development stages as a guide.
```

Comment:
if youre gonna number sources like this, it's worth having a list of sources at the bottom of the page that say what each one is